packages/office-ui-fabric-react/src/components/DetailsList/DetailsHeader.tsx

```diff
@@ -115,65 +123,76 @@ export class DetailsHeader extends BaseComponent<IDetailsHeaderProps, IDetailsHe
         }) }
         ref='root'
         onMouseMove={ this._onRootMouseMove }
-        data-automationid='DetailsHeader'>
-        <FocusZone ref='focusZone' direction={ FocusZoneDirection.horizontal }>
-          { (selectAllVisibility !== SelectAllVisibility.none) ? (
-            <div className={ css('ms-DetailsHeader-cellWrapper', styles.cellWrapper) } role='columnheader'>
-              <button
-                type='button'
-                className={ css('ms-DetailsHeader-cell is-check', styles.cell, styles.cellIsCheck) }
-                onClick={ this._onSelectAllClicked }
-                aria-label={ ariaLabelForSelectAllCheckbox }
-                aria-pressed={ isAllSelected }
-              >
-                <Check checked={ isAllSelected } />
-              </button>
-            </div>
-          ) : null }
-          { groupNestingDepth > 0 ? (
-            <button
-              type='button'
-              className={ css('ms-DetailsHeader-cell', styles.cell) }
-              onClick={ this._onToggleCollapseAll }>
-              <Icon
-                className={ css(
-                  'ms-DetailsHeader-collapseButton',
-                  styles.collapseButton,
-                  isAllCollapsed && ('is-collapsed ' + styles.collapseButtonIsCollapsed)
-                ) }
-                iconName='ChevronDown'
-              />
-            </button>
-          ) : (null) }
-          { GroupSpacer({ count: groupNestingDepth - 1 }) }
-          { columns.map((column, columnIndex) => (
-            <div
+        data-automationid='DetailsHeader'
+        direction={ FocusZoneDirection.horizontal }
+      >
+        { (selectAllVisibility !== SelectAllVisibility.none) ? (
+          <FocusZone
+            direction={ FocusZoneDirection.horizontal }
+            className={ css('ms-DetailsHeader-cell', 'ms-DetailsHeader-cellIsCheck', styles.cell, styles.cellIsCheck, checkStyles.owner, {
```

Comment:
I need to have the `:focus` state of the root element impact the check visiblity. So it's best to apply a `Check`-specific style to `DetailsHeader` rather than have the `DetailsHeader` try to override the styles.